packages/ui-components/src/components/DataTable/useDataTableExport.js

```diff
@@ -9,14 +9,16 @@ import { useTable } from 'react-table';
 import moment from 'moment';
 
 export const useDataTableExport = (columns, data, title) => {
-  const { rows: tableData, columns: tableColumns } = useTable({
+  const { headerGroups, rows: tableData, columns: tableColumns } = useTable({
     columns,
     data,
   });
 
   const doExport = () => {
     // Get data from react table properties
-    const header = [tableColumns.map(col => col.Header)];
+    const header = headerGroups.map(({ headers }) =>
+      headers.map(({ Header: getHeader, id }) => getHeader({ column: { id } })),
```

Comment:
Could you make this polymorphic, so that it can handle `Header` being a string or a function? In the style of e.g. https://github.com/beyondessential/tupaia/blob/dev/packages/ui-components/src/components/Table/TableRow.js#L32

Reply:
Sounds good!